README.md

```diff
@@ -1,33 +1,42 @@
-CMPUT404-project-socialdistribution
-===================================
+# Social Distribution
 
 CMPUT404-project-socialdistribution
 
 See project.org (plain-text/org-mode) for a description of the project.
 
 Make a distributed social network!
 
-Contributing
-============
+# Team Members 
+The team members and their GitHub usernames
+```
+Olivier Vadiavaloo : olivier-vadiaval
+Jawad Hossain : jawadhossain
+Diego Becerra : debecerra
+Tyler Elston : tylerelston
+Michelle Wang : mxiuwang
+```
+
+# Contributing
 
 Send a pull request and be sure to update this file with your name.
 
-Contributors / Licensing
-========================
+# Contributors / Licensing
+
 
 Generally everything is LICENSE'D under the Apache 2 license by Abram Hindle.
```

Comment:
Should our names be included here since it will be covering our code too?

Reply:
I think it would make sense to add it there too.